src/plugins/expressions/public/loader.ts

```diff
@@ -64,9 +72,14 @@ export class ExpressionLoader {
       this.render(data);
     });
 
+    this.render$.subscribe(() => {
+      this.loadingSubject.next(false);
```

Comment:
are we emitting false now after loading has completed ?

Reply:
No, I left the public observable the same, 
This is done due to:
```
this.loading$ = this.loadingSubject.asObservable().pipe(
       filter(_ => _ === true),
       map(() => void 0)
     );
```